angrmanagement/data/instance.py

```diff
@@ -37,7 +37,7 @@ class Instance:
     """
     An object to give access to normal angr project objects like a Project, CFG, and other analyses.
     """
-    project: Union[angr.Project, ObjectContainer]
+    project: ObjectContainer
```

Comment:
Presumably the annotations were like this for code completion. We should be consistent in style though; that is, either switch all of these lines over to annotate only ObjectContainer, or do the Union with the real type for completion. I'll defer to @ltfish on it